src/altinn-app-frontend/src/components/summary/AttachmentWithTagSummaryComponent.test.tsx

```diff
@@ -30,7 +29,7 @@ describe('AttachmentWithTagSummaryComponent', () => {
   const extendedState = {
     attachments: {
       attachments: {
-        [typeName]: [
```

Comment:
Yep, I went a bit overboard when getting this to work. Reverting back to using a const here. Since it is a const, the inferred type is `"FileUploadWithTag"`, not `string`, so it works perfectly fine that way as well.